mail/fdm/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nicm fdm 2.1
+checksums           rmd160  b229c09c246f659219dc9586552f5eadcb5da93f \
+                    sha256  c40afa3c2ca1f61d5eb5bb7ebd4befd11e2956ff4b15efa0f27cbed744e3b738 \
+                    size    184748
+revision            0
+
+platforms           darwin
+categories          mail
+license             ISC
+maintainers         {shuxiao9058 @shuxiao9058} openmaintainer
+description         A program designed to fetch mail.
+long_description \
+   fdm is a program designed to fetch mail from POP3 or IMAP servers, \
+   or receive local mail from stdin, and deliver it in various ways.
+homepage            https://github.com/nicm/fdm
+
+compiler.c_standard   1999
+compiler.cxx_standard 2017
+
+depends_build-append \
+    port:pkgconfig
+
+depends_lib-append \
+    port:tdb \
+    port:openssl \
```

Comment:
You can actually use the `openssl` portgroup instead of the specific `openssl` port.  Just add `PortGroup openssl 1.0` underneath the entry for the `github` portgroup above.